src/drivers/mct/cime_config/buildnml

```diff
@@ -23,6 +23,24 @@ from CIME.XML.files import Files
 logger = logging.getLogger(__name__)
 
 ###############################################################################
+def _get_time_in_seconds(time, unit):
```

Comment:
I could not find such a function so I moved this function from buildnml to CIME.utils.